integration-tests/suites/base.go

```diff
@@ -186,6 +186,12 @@ func (s *IntegrationTestSuiteBase) execContainer(containerName string, command [
 	return s.executor.Exec(cmd...)
 }
 
+func (s *IntegrationTestSuiteBase) execContainerShellScript(containerName string, script string, args ...string) (string, error) {
+	cmd := []string{"docker", "exec", "-i", containerName, "/bin/sh", "-s"}
```

Comment:
With the new podman support on master this will need to be:

```suggestion
	cmd := []string{common.RuntimeCommand, "exec", "-i", containerName, "/bin/sh", "-s"}
```